### PR TITLE
Remove yarn resolutions

### PR DESCRIPTION
**Pull request by fayi-da**
Part of #13383.

### CODE REVIEW

**Review by S11001001 — APPROVED** [review]
Questions before merging:

* [x] `yarn audit` still passes in both directories?
* [ ] Does Navigator front-end still work according to a quick smoke-test in this branch?

**Comment by stefanobaghino-da** [comment]
> Questions before merging:
> 
> * [ ]  `yarn audit` still passes in both directories?
> * [ ]  Does Navigator front-end still work according to a quick smoke-test in this branch?

Could be worth running `yarn audit` automatically as part of automated tests to make sure this question doesn't need to be answered manually.

**Comment by stefanobaghino-da** [comment]
> @ray-roestenburg-da I think I an only merge this after your approval

You don't have to, you only need one approval to merge. We use the `CODEOWNERS` for visibility, not to mandate certain approvals. Any other merge requirements are handled informally.

**Pull request merged**
